src/eeprom/types.rs

```diff
@@ -262,10 +262,14 @@ impl From<PdoType> for CategoryType {
 #[cfg_attr(feature = "defmt", derive(defmt::Format))]
 #[repr(u8)]
 pub enum FmmuUsage {
+    /// TODO
```

Comment:
I'll obviously document these properly if you agree with exposing these types publicly.